jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1515,94 +1999,47 @@ Function Start-Migration {
             break
         }
 
-        # Conditional ParameterSet logic
-        If ($PSCmdlet.ParameterSetName -eq "form") {
-            $SelectedUserName = $inputObject.SelectedUserName
-
-            $JumpCloudUserName = $inputObject.JumpCloudUserName
-            $TempPassword = $inputObject.TempPassword
-            if (($inputObject.JumpCloudConnectKey).Length -eq 40) {
-                $JumpCloudConnectKey = $inputObject.JumpCloudConnectKey
-            }
-            if (($inputObject.JumpCloudAPIKey).Length -eq 40) {
-                $JumpCloudAPIKey = $inputObject.JumpCloudAPIKey
-                $ValidatedJumpCloudOrgID = $inputObject.JumpCloudOrgID
-            }
-            $InstallJCAgent = $inputObject.InstallJCAgent
-            $AutobindJCUser = $inputObject.AutobindJCUser
-
-            if ($AutoBindJCUser -eq $true) {
-                # Throw error if $ret is false, if we are autobinding users and the specified username does not exist, throw an error and terminate here
-                $ret, $JumpCloudUserId, $JumpCloudUsername, $JumpCloudsystemUserName = Test-JumpCloudUsername -JumpCloudApiKey $JumpCloudAPIKey -JumpCloudOrgID $ValidatedJumpCloudOrgID -Username $JumpCloudUserName
-                # Write to log all variables above
-                Write-ToLog -Message:("Test-JumpCloudUsername Results:`nUserFound: $($ret)`nJumpCloudUserName: $($JumpCloudUserName)`nJumpCloudUserId: $($JumpCloudUserId)`nJumpCloudsystemUserName: $($JumpCloudsystemUserName)")
-
-                if ($JumpCloudsystemUserName) {
-                    $JumpCloudUsername = $JumpCloudsystemUserName
-                }
-                if ($ret -eq $false) {
-                    Write-toLog ("The specified JumpCloudUsername does not exist")
-                    break
-                }
-            }
+        $hostname = hostname # Get computer hostname
```

Comment:
Two things here. I feel like this code is trying to do the same thing as line [198](https://github.com/TheJumpCloud/jumpcloud-ADMU/pull/125/files#diff-4928a72602696edc556d72f323c65a31f1ebeccd783de4d21e823bcebb2b5187R198)

So actually this should be reviewed to see if we can just add it to that function.

Also just to keep things consistent I'd rather see us use `$env:computername` instead of `hostname` here. I get that hostname is a command that's been added to powershell but reading it looks like it's an undefined variable.

```suggestion
        # Get computer hostname
        $hostname =  $env:computername
```
![Screenshot 2024-07-03 at 8 08 32 AM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/ec3ac3e5-cacf-43ba-997d-aba7a50e0164)